enhancements/single-node/single-node-openshift-day2-workers.md

```diff
@@ -0,0 +1,290 @@
+---
+title: single-node-openshift-day2-workers
+authors:
+  - "@omertuc"
+reviewers:
+  - "@romfreiman"
+  - "@eranco74"
+  - "@tsorya"
+  - TBD
+approvers:
+  - TBD
+api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers)
+  - TBD
+creation-date: 2021-01-05
+last-updated: 2021-01-05
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/MGMT-8414
+see-also:
+  - "https://github.com/openshift/enhancements/tree/master/enhancements/single-node"
+  - "https://github.com/openshift/enhancements/blob/master/enhancements/installer/single-node-installation-bootstrap-in-place.md"
+replaces:
+superseded-by:
+---
+
+# Single Node OpenShift Day-2 Workers
+
+## Summary
+
+This enhancemnet aims to enable adding workers to a single-node cluster by
+dealing with a "floating ingress" issue encountered in UPI-installed single
```

Comment:
is it only "floating ingress" issue? other operators are fine with it? Shouldn't we just set all the "control-plane" operators to run on master in case of SNO?

Reply:
If other operators get pods on the worker node, why would that be a problem?